c2corg_ui/templates/helpers/view.html

```diff
@@ -173,10 +197,10 @@ <h3 class="heading show-phone" ng-click="mainCtrl.animateHeaderIcon($event)" dat
   % if text == 'weather':
```

Comment:
Using ``text`` as variable name is confusing since it's more a type than a description of what the variable is. ``attr`` or ``attr_name`` would have been clearer I think.